### DISCUSSION TITLE
caching-docker-layers-on-serverless-build-hosts-with-multi-stage-builds---target,-and---cache-from/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#72] [Announcements]
# caching-docker-layers-on-serverless-build-hosts-with-multi-stage-builds---target,-and---cache-from/

In this post I describe how I improved the build performance of ASP.NET Core Docker containers built using AWS CodeBuild by using --target and --cache-from.

https://andrewlock.net/caching-docker-layers-on-serverless-build-hosts-with-multi-stage-builds---target,-and---cache-from/

**Comment by andresanies** [comment]
Awesome article, thank you for your detailed explanation of the usage of `--cache-from` option.

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at July 15 2019, 10:11, in reply to [Aaron Roydhouse](https://disqus.com/by/aaronroydhouse/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/72#discussioncomment-6054167)</em>

---

<p>Interesting! I'm actually not sure if it will matter in this case though - the most important cache is the `builder` image (which contains the build and restore artefacts). Also there's no shared layers between builder and latest, so I think the correct image will be selected for cache in this case.</p>

[a comment by an automated account is not shown]
